Code/src/main/java/nl/utwente/viskell/ui/TouchDisplay.java

```diff
@@ -32,16 +35,21 @@ public TouchDisplay(double x, double y, double radius, int touchId) {
         label = new Label();
         label.textProperty().bindBidirectional(labelText);
         label.setTextFill(COLOR_OPAQUE);
-        label.relocate(0, radius * 2);
+        label.relocate(0, touchCircleRadius * 2);
         getChildren().add(label);
+        
+        line = new Line(x, y, bounds.getMinX(), bounds.getMinY());
+        getChildren().add(line);
 
         setBackground(new Background(new BackgroundFill(Color.TRANSPARENT, null, null)));
     }
 
-    @Override
-    public void relocate(double x, double y) {
+    public void relocate(double x, double y, Bounds bounds) {
```

Comment:
This no longer overrides `relocate`. Would calling the non-overridden `relocate` do subtly the wrong thing? If so, I would be in favor of renaming this method.